feature-libs/product-configurator/rulebased/components/attribute/types/multi-selection-image/configurator-attribute-multi-selection-image.component.html

```diff
@@ -54,8 +42,13 @@
           <div *ngIf="!getImage(value)" class="cx-img-dummy"></div>
           {{ getLabel(expMode, value.valueDisplay, value.valueCode) }}
           <cx-configurator-price
+            *ngIf="!isAsyncPricing"
             [formula]="extractValuePriceFormulaParameters(value)"
           ></cx-configurator-price>
+          <cx-configurator-price-async
```

Comment:
Please, fix styling. The price should be centered: 
![image](https://github.com/SAP/spartacus/assets/61147963/25fb9210-6016-4b62-a293-fd9771628b58)

It should be fix in _configurator-attribute-selection-image.scss mixin, see line 102. This line works only for the old component but not for the new one.

Reply:
fixed